Server/mods/deathmatch/logic/CAccount.h

```diff
@@ -101,6 +101,7 @@ class CAccount
     bool                       AuthorizeSerial(const SString& strSerial, const SString& strWho);
     bool                       RemoveSerial(const SString& strSerial);
     void                       RemoveUnauthorizedSerials();
+    void                       SetAccountSerial(const SString strSerial);
```

Comment:
SString is no longer used, please use std::string instead

Reply:
adjusted 62ebf06